frontend/src/premium/eagle-eye/components/list/eagle-eye-settings.vue

```diff
@@ -72,6 +72,12 @@
   </div>
 </template>
 
+<script>
+export default {
+  name: 'CrEagleEyeSetting'
```

Comment:
Could you delete this one for now maybe? While we are not using a new convention?